README.md

```diff
@@ -20,13 +20,14 @@ Quick Links:
 ## How does it work?
 
 Cloud Run Release Manager periodically checks for new revisions in the services
-that opted-in for gradual rollouts. If a new revision with no traffic is found,
-the Release Manager automatically assigns it some initial traffic. This new
-revision is labeled `candidate` while the previous revision serving traffic is
-labeled `stable`.
+that opted-in for gradual rollouts. To opt-in services for gradual rollouts, a
+label must be assigned to them. If a new revision with no traffic is found in
+these services, the Release Manager automatically assigns some initial traffic
+to the new revision. This new revision is tagged `candidate` while the previous
+revision serving traffic is tagged `stable`.
 
-Depending on the candidate's health, traffic to the `candidate` is increased
-or traffic to the candidate is dropped and is redirected to the `stable` revision.
+Depending on the candidate's health, traffic to the `candidate` is increased or
+traffic to the candidate is dropped and is redirected to the `stable` revision.
```

Comment:
this doesn't entirely capture the situation.

you can say we always manage 2 revisions, stable is the one that reached 100%, the other one is candidate which is the newest deployment. [...]

Reply:
My guess is that this comment is about lines 26-27. Is that so?

Reply:
> Depending on the candidate's health, traffic to the `candidate` is increased or
> traffic to the candidate is dropped and is redirected to the `stable` revision.

it was about this part.